com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/BloomUpsample.compute

```diff
@@ -42,8 +42,5 @@ void KMain(uint3 dispatchThreadId : SV_DispatchThreadID)
 
     float3 output = lerp(highRes, lowRes, Scatter);
 
-    // Guard bands
-    output *= all(dispatchThreadId.xy < uint2(_TexelSize.xy));
```

Comment:
This is gone with a recent PR that fixed a bug, please remove from this PR to avoid conflicts.